packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -335,6 +337,14 @@ export default class TableResize implements EditorPlugin {
 
     private startVerticalResizeTable = (e: MouseEvent) => {
         this.resizingState = ResizeState.Vertical;
+
+        const vtable = new VTable(this.currentTd);
+        if (vtable) {
+            const rect = normalizeRect(this.currentTd.getBoundingClientRect());
+            this.currentCellsToResize = vtable.getCellsWithBorder(rect.right, true);
```

Comment:
add inline comment for constant parameters

Reply:
done